### PR TITLE
Add software probe for OpenWrt

### PR DESCRIPTION
**Pull request by ja-pa**
This PR adds support for the software probe to OpenWrt. Right now it's similar to the Turris software probe, but it will probably diverge in the future. 

After merging this code I want to send the package to OpenWrt packages repository which will point to this code.

Signed-off-by: Jan Pavlinec <jan.pavlinec@nic.cz>

### CODE REVIEW

File: bin/arch/openwrt-sw-probe/openwrt-sw-probe-ATLAS.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+KILL_TELNETD_CMD=kill_telnetd
+MOUNT_FS_CMD=:
+SETUP_NETWORK_CMD=:
+NTPCLIENT_CMD=:
+SU_CMD=""
+CHOWN_FOR_MSM=:
+CHMOD_FOR_MSM=chmod_for_msm
+SET_HOSTNAME=:
+
+# For OpenWRT we need telnetd to run as root.
```
Comment (BKPepe) [inline bin/arch/openwrt-sw-probe/openwrt-sw-probe-ATLAS.sh:25]:
```suggestion
# For OpenWrt we need telnetd to run as root.
```

**Comment by BKPepe** [comment]
@PhilipHomburg Can you please look at this pull request?

**Comment by phicoh** [comment]
Sorry that I completely forgot to merge this. I merge the pull request into the devel branch and fixed the spelling of OpenWrt.

**Comment by Ansuel** [comment]
@ja-pa can you post the pr about the openwrt package on the openwrt repo?

**Comment by ja-pa** [comment]
> @ja-pa can you post the pr about the openwrt package on the openwrt repo?

Yes, PR is here https://github.com/openwrt/packages/pull/15328. It is still a draft because of a compilation error.

**Pull request merged**